Guide to help people setup a start-many capable wallet

perrywoodin opened the pull request:
Wrote a brief start-many setup guide with the help of UdjinM6 and he suggested I add it to the doc directory.

UdjinM6 commented:
Good stuff!

UdjinM6 commented on doc/guide-startmany.md:
```diff
@@ -0,0 +1,151 @@
+
+The wallet will re-scan and you will see your available balance increase by the amount that was in the imported wallet.
+
+Skip Option 2. and go to Create masternode.conf file
```
should go to "Create New Masternode Private Keys" from here

UdjinM6 commented on doc/guide-startmany.md:
```diff
@@ -0,0 +1,151 @@
+
+Just like setting up a standard MN. Send exactly 1,000 DRK to each new address created above.
+
+### Create New Keys
```
should be 
## Create New Masternode Private Keys

UdjinM6 commented on doc/guide-startmany.md:
```diff
@@ -0,0 +1,151 @@
+
+Create the masternode.conf file in the same directory as your wallet.dat.
+
+Copy the private key and correspondig collateral output transaction that holds the 1K DRK.
```
This part should be changed: "private key" -> "masternode private key"

UdjinM6 commented on doc/guide-startmany.md:
```diff
@@ -0,0 +1,151 @@
+
+Copy the private key and correspondig collateral output transaction that holds the 1K DRK.
+
+The private key may be an existing key from Option 1, or a newly generated key from Option 2. 
```
This is completely wrong in terms of security. Only Masternode Private Keys should be used in masternode.conf

UdjinM6 commented on doc/guide-startmany.md:
```diff
@@ -0,0 +1,151 @@
+
+## Update darkcoin.conf on server
+
+If you generated a new private key, you will need to update the remote darkcoin.conf files.
```
This part should be changed: "private key" -> "masternode private key"

UdjinM6 commented on doc/guide-startmany.md:
````diff
@@ -0,0 +1,151 @@
+```sudo nano .darkcoin/darkcoin.conf```
+
+### Edit the masternodeprivkey
+If you generated a new private key, you will need to update the masternodeprivkey value in your remote darkcoin.conf file.
````
This part should be changed: "private key" -> "masternode private key"

UdjinM6 commented:
It must be as clear as possible that it's not private key that you have controls your coins used here, it's some dummy one-time only "private key". Need to be fixed.

perrywoodin commented:
UdjinM6. Thanks for the additional review and comments. I have incorporated your suggestions and added a note about the importance of keeping the wallet private key out of the masternode.conf

UdjinM6 commented:
Good job @perrywoodin !

freynder commented on doc/guide-startmany.md:
```diff
@@ -0,0 +1,153 @@
+
+Open your QT Wallet and go to console (from the menu select Tools => Debug Console)
+
+Issue the following:
```
Please make it clear a seperate masternode key is needed for each masternode. People may think they need to generate it only once and run into trouble later.

freynder replied:
Very good guide indeed, thank you!

perrywoodin replied:
Thanks for the comment. I have added a note about creating a masternode private key for each MN.

UdjinM6 replied:
Good find @freynder ! Having the same masternode private key for many masternodes does not break security in any way but if someone break into one of your masternodes he can stop all of your masternodes using the same masternode private key which can be an issue. So, yes, it's better to have a separate masternode private key for each masternode and worth mentioning for sure.

freynder replied:
Not only that, but it is also used by the masternode to detect if it should start:
https://github.com/darkcoin/darkcoin/blob/master/src/masternode.cpp#L171
It will also become more important with the dsee/dseep rewrite I'm currently working on.

freynder commented:
Very good guide indeed, thank you!

evan82 commented:
Very nice, good work everyone :+1:
